src/main/java/aquality/appium/mobile/actions/TouchActions.java

```diff
@@ -0,0 +1,42 @@
+package aquality.appium.mobile.actions;
+
+import aquality.appium.mobile.application.AqualityServices;
+import aquality.appium.mobile.configuration.ISwipeConfiguration;
+import aquality.selenium.core.utilities.IElementActionRetrier;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.touch.offset.PointOption;
+import org.openqa.selenium.Point;
+import java.util.function.UnaryOperator;
+import static io.appium.java_client.touch.WaitOptions.waitOptions;
+
+public class TouchActions implements ITouchActions {
+
+    @Override
+    public void swipe(Point startPoint, Point endPoint) {
+        AqualityServices.getLocalizedLogger().info(
+                "loc.action.swipe",
+                startPoint.getX(),
+                startPoint.getY(),
+                endPoint.getX(),
+                endPoint.getY());
+        performTouchAction(touchAction -> touchAction.press(PointOption.point(startPoint))
+                .waitAction(waitOptions(AqualityServices.get(ISwipeConfiguration.class).getTimeout())), endPoint);
```

Comment:
will ISwipeConfiguration contain only properties related to swipes all perhabs better to get there full list of properties of touch actions